README.md

```diff
@@ -70,11 +74,103 @@ COPY root/ /
 
 For most users this will suffice and anything in the root/ folder of the repository will be added to the end users Docker container / path.
 
-The most common paths to leverage for Linuxserver images will be:
+#### Legacy (v2) mods
```

Comment:
Can we move this below the new v3 info? It makes sense to see new first and legacy second.